### PR TITLE
[14.0][FIX] purchase_allowed_product: better inheritance

### PR DESCRIPTION
**Pull request by GSLabIt**

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

For some reason, with multiple purchase modules installed, the setting added in https://github.com/OCA/purchase-workflow/pull/2088 was not visible. I could not reproduce the issue but it's present in current runboat

File: purchase_allowed_product/README.rst [hunk 1]
```diff
@@ -74,7 +74,10 @@ Contributors
 
   * Darius Žižys
 
-* Ooops404 <info@ooops404.com>
+* `Ooops404 <https://www.ooops404.com>`__:
+  * Ooops404 <info@ooops404.com>
```
Comment (pedrobaeza) [inline purchase_allowed_product/README.rst]: This is not a contributor per se, so if you put the company name, don't put this line.

Comment (francesco-ooops) [reply purchase_allowed_product/README.rst]: @pedrobaeza https://github.com/OCA/maintainer-tools/issues/592

@GSLabIt I think if you remove line 78 it should be good to go

Comment (pedrobaeza) [reply purchase_allowed_product/README.rst]: Exactly

**Comment by francesco-ooops** [comment]
@pedrobaeza contributors fixed, can we merge? thanks!

**Pull request closed without merging**